package.json

```diff
@@ -69,6 +68,7 @@
     ]
   },
   "dependencies": {
-    "classnames": "^2.2.5"
+    "classnames": "^2.2.5",
+    "node-uuid": "^1.4.8"
```

Comment:
`uuid` had a dependency on native nodejs module `crypto`. This won't work on front-end. `node-uuid` has no dependencies and a lot of GH stars.